br-ext/patches/xen/0001-Revert-xen-arm-Restrict-the-amount-of-memory-that-do.patch

```diff
@@ -0,0 +1,37 @@
+From 82a8e6a916f1a7bb3910c365197319b37085d58c Mon Sep 17 00:00:00 2001
```

Comment:
Would it make sense to take this patch in a separate commit?

Reply:
Can't it be added before the "qemu_v8: build Xen with Buildroot" commit?

Reply:
Yes, I will split the commit.